File: CS50_assignments/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height : ");
    }
    while(height < 1 || height > 8);

    for(int i = 0; i < height; i++)
    {
        int c = i, r = i;
        int space = height - (i + 1);

        while(space > 0)
        {
            printf("%c",' ');
            space--;
        }

        while( c >= 0)
        {
            printf("%c",'#');
            c--;
        }
        printf("%c%c", ' ',' ');
        while( r >= 0)
        {
            printf("%c",'#');
            r--;
        }
        printf("\n");
    }
}

File: CS50_assignments/week6/sentimental-mario-less/mario.py
from cs50 import get_int

def main():
    height = get_height()
    for i in range(height):
        print(" "*(height-(i+1)),end="")
        for j in range(i+1):
            print("#", end="")
        print()

def get_height():
     while(True):
        n = get_int("height:")
        if(n>0 and n < 9):
            break
     return n

main()